File: src/crowstream/user/profile/typeDefs.js
export const profileTypeDef = `
    type Profile {
        id: String!
        account_id: String
        name: String!
    }
    input ProfileInput {
        name: String!
    }
    type Profiles {
        account_id: String!
        profiles: [Profile]!
    }
`;

export const profileQueries = `
    allProfilesByUser: Profiles!
    profileById(id: String!): Profile!
`;

export const profileMutations = `
    createProfile(profile: ProfileInput!): Profile!
`;

File: api_gateway/schema/user/profile/type_defs.py
import graphene

class Profile(graphene.ObjectType):
    id = graphene.NonNull(graphene.String, name='id')
    account_id = graphene.String(name='account_id')
    name = graphene.NonNull(graphene.String, name='name')

class ProfileInput(graphene.InputObjectType):
    name = graphene.NonNull(graphene.String, name='name')

class Profiles(graphene.ObjectType):
    account_id = graphene.NonNull(graphene.String, name='account_id')
    profiles = graphene.List(Profile, name='profiles')